_attributes/common-attributes.adoc

```diff
@@ -14,7 +14,7 @@
 //gitops
 :gitops-title: Red Hat OpenShift GitOps
 :gitops-shortname: GitOps
-:gitops-ver: 1.15
+:gitops-ver: 1.17
 :upstream-ver: 2.13
```

Comment:
Upstream version here I think we are talking about Argo CD version. If that is the case, we need to put it as 3.0. Not sure where it is referenced. But it is better to update this.

Reply:
Fixed the upstream version to 3.0.

Reply:
LGTM